File: src/dependent/fulfillment/order.py
from .address import Address


class Order:
    def __init__(self, order_id: str, product_id: str, quantity: int, address: Address) -> None:
        self._order_id = order_id
        self._product_id = product_id
        self._quantity = quantity
        self._address = address

    @property
    def order_id(self) -> str:
        return self._order_id

    @property
    def product_id(self) -> str:
        return self._product_id

    @property
    def quantity(self) -> int:
        return self._quantity

    @property
    def address(self) -> Address:
        return self._address

File: src/dependent/fulfillment/Order.java
package org.example.dependent.fulfillment;

public class Order {
    private final String orderId;
    private final String productId;
    private final int quantity;
    private final Address address;

    public Order(String orderId, String productId, int quantity, Address address) {
        this.orderId = orderId;
        this.productId = productId;
        this.quantity = quantity;
        this.address = address;
    }

    public String getOrderId() {
        return orderId;
    }

    public String getProductId() {
        return productId;
    }

    public int getQuantity() {
        return quantity;
    }

    public Address getAddress() {
        return address;
    }
}


